deploy/kustomize/base/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - 01_namespace.yaml
+  - 02_secret.yaml
+  - 03_operator_source.yaml
+  - 04_catalog_source_config.yaml
```

Comment:
with CONTENT_ONLY mode ( https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/deploy/deploy_marketplace.sh#L21 ) we have a mechanism where we prepare everything for QE, but we let them manually trigger the deployment from UI to check what is going to happen there.
In that case the namespace and the operator group are supposed to be created by the console (see also https://github.com/kubevirt/hyperconverged-cluster-operator/issues/476 ).
Can we mimic something like that?

Reply:
New behavior:
If we set CREATE_NAMESPACE to false prior running `deploy_kustomize.sh`, the namespace is not being created.
This flag is taken into account only if CONTENT_ONLY is true.